packages/management-api-app/src/bootstrap-modules.ts

```diff
@@ -0,0 +1,18 @@
+export async function bootstrapModules() {
```

Comment:
return type

Reply:
`node` vs `bundler` - I think I have my opinion shaped after https://www.typescriptlang.org/tsconfig/#moduleResolution:

> 'bundler' for use with bundlers. Like node16 and nodenext, this mode supports package.json "imports" and "exports", but unlike the Node.js resolution modes, bundler never requires file extensions on relative paths in imports.

and `node` is an alias for `node10`:

> 'node10' (previously called 'node') for Node.js versions older than v10, which only support CommonJS require. You probably won’t need to use node10 in modern code.